SmartDeviceLink/SDLStreamingMediaLifecycleManager.m

```diff
@@ -434,6 +442,10 @@ - (void)didEnterStateVideoStreamReady {
     }
 }
 
+- (void)didEnterStateVideoStreamSuspended {
```

Comment:
This method is unused

Reply:
True. I wasn‘t sure if the state methods *must* be implemented. I can remove it if it‘s not necessary.

Reply:
@NicoleYarroch I added the requested suspended notification to the method.